.env.local.example

```
@@ -1,7 +1,14 @@
-# Mapbox MCP Server Credentials
-# Replace with your actual Smithery profile ID and API key
-SMITHERY_PROFILE_ID="your_smithery_profile_id_here"
-SMITHERY_API_KEY="your_smithery_api_key_here"
+# Composio Mapbox Integration
+# Replace with your actual Composio auth config ID and user ID
+COMPOSIO_MAPBOX_AUTH_CONFIG_ID="ac_YOUR_MAPBOX_CONFIG_ID"
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)